backend/x/gorgonnx/conv.go

```diff
@@ -108,6 +83,39 @@ func (c *conv) apply(g *Graph, ns ...*Node) error {
 	return nil
 }
 
+// autopadding needs to be applied now because it needs to be aware of the shape of the nodes
+func (c *conv) autopadding(children []*Node) error {
```

Comment:
Cool; I will add tests for this function soon.
many thanks, once again this really helps